File: Python/level 1/Outputting triangles.py
def printTriangle(num):
    s=""
    for a in range(0,num):
        s+="*"*(a+1)+"\n"
    return s


# 아래는 테스트로 출력해 보기 위한 코드입니다.
print( printTriangle(3) )

File: Javascript/level 1/Outputting triangles.js
function printTriangle(num) {
  var result = ''
  // 함수를 완성하세요
  for(var a=0; a<num; a++)
    result+='*'.repeat(a+1)+'\n';
  return result
}


// 아래는 테스트로 출력해 보기 위한 코드입니다.
console.log( printTriangle(3) );